db/migrate/20230104181758_rename_booking_date_columns.rb

```diff
@@ -0,0 +1,6 @@
+class RenameBookingDateColumns < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :bookings, :start_date, :pick_up_date
+    rename_column :bookings, :end_date, :return_date
+  end
+end
```

Comment:
Good change but how about just simply changing the field names of the previous migrations /👍